ui-grid/ui-grid.d.ts

```diff
@@ -1064,6 +1074,15 @@ declare module uiGrid {
         (scrollEvent: JQueryMouseEventObject): void;
     }
 
+    export interface sortChangedHandler {
+        /**
+         * Sort change event callback
+         * @param {IGridInstance} grid instance
+         * @param {IGridColumn} array of gridColumns that have sorting on them, sorted in priority order
+         */
+        (grid: IGridInstance, columns: IGridColumn[]): void;
```

Comment:
I know this PR was opened before #5972 was merged, but I think we should use the generic versions of `IGridInstance` and `IGridColumn` here to be consistent and provide type helps when using this feature.
In order to do this, change: 
`uiGrid.IGridInstance` to `uiGrid.IGridInstanceOf<TEntity>`,
`uiGrid.IGridColumn` to `uiGrid.IGridColumnOf<TEntity>`,
`sortChangedHandler` to `sortChangedHandler<TEntity>`,
then add generics up the chain as necessary.  Let me know if you have any questions.